docs/source/guides/session_limits.rst

```diff
@@ -0,0 +1,87 @@
+How To Set Session Limits
+=========================
+
+What is the session limit for?
+*****************************
+
+By default an orchestrator can handle a maximum of **10** incoming livestreams at a time. Whenever the number of concurrent transcoding sessions goes above this, the orchestrator returns an ``OrchestratorCapped`` error to the broadcaster.
+
+Of course, the transcoding capacity and the bandwidth varies for everyone - so you can set the ``-maxSessions`` CLI parameter with a value that reflects available resources to maximize work received from broadcasters.
+
+For example if you have enough transcoding capacity and bandwidth to handle ``30`` streams, you can set the max sessions using the following command (other flags omitted):
+
+::
+
+    $ livepeer -orchestrator -transcoder -maxSessions 30
+
+How to calculate my session limit? 
+*********************************
+
+Calculate the session limit due to (1) transcoding hardware and (2) bandwidth as explained below, and take the minimum of the two. Finally, pass it through the ``-maxSessions`` parameter to your node as explained above.
```

Comment:
I'd go with "based on" instead of "due to" as the latter is more used to describe a causal relationship between A and B, similar to "because of".